[2025-02-01T01:37:09.176+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: naver_flight_bigquery_pipeline.process_flight_data manual__2025-02-01T01:30:09.552178+00:00 [queued]>
[2025-02-01T01:37:09.182+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: naver_flight_bigquery_pipeline.process_flight_data manual__2025-02-01T01:30:09.552178+00:00 [queued]>
[2025-02-01T01:37:09.182+0000] {taskinstance.py:1308} INFO - Starting attempt 3 of 3
[2025-02-01T01:37:09.193+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): process_flight_data> on 2025-02-01 01:30:09.552178+00:00
[2025-02-01T01:37:09.199+0000] {standard_task_runner.py:57} INFO - Started process 461 to run task
[2025-02-01T01:37:09.201+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'naver_flight_bigquery_pipeline', 'process_flight_data', 'manual__2025-02-01T01:30:09.552178+00:00', '--job-id', '103', '--raw', '--subdir', 'DAGS_FOLDER/json_to_bigquery.py', '--cfg-path', '/tmp/tmpbuvm677e']
[2025-02-01T01:37:09.203+0000] {standard_task_runner.py:85} INFO - Job 103: Subtask process_flight_data
[2025-02-01T01:37:09.233+0000] {task_command.py:410} INFO - Running <TaskInstance: naver_flight_bigquery_pipeline.process_flight_data manual__2025-02-01T01:30:09.552178+00:00 [running]> on host 3f33197bb456
[2025-02-01T01:37:09.348+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='naver_flight_bigquery_pipeline' AIRFLOW_CTX_TASK_ID='process_flight_data' AIRFLOW_CTX_EXECUTION_DATE='2025-02-01T01:30:09.552178+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-01T01:30:09.552178+00:00'
[2025-02-01T01:37:09.355+0000] {base.py:73} INFO - Using connection ID 'google_cloud_default' for task execution.
[2025-02-01T01:37:11.492+0000] {base.py:73} INFO - Using connection ID 'google_cloud_default' for task execution.
[2025-02-01T01:37:11.537+0000] {taskinstance.py:1824} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 181, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 198, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/json_to_bigquery.py", line 102, in upload_to_bigquery
    client.load_table_from_dataframe(df, table_ref, job_config={"schema": schema})
  File "/home/airflow/.local/lib/python3.7/site-packages/google/cloud/bigquery/client.py", line 2602, in load_table_from_dataframe
    _verify_job_config_type(job_config, LoadJobConfig)
  File "/home/airflow/.local/lib/python3.7/site-packages/google/cloud/bigquery/_helpers.py", line 951, in _verify_job_config_type
    job_config=job_config,
TypeError: Expected an instance of LoadJobConfig class for the job_config parameter, but received job_config = {'schema': [{'name': 'date', 'type': 'STRING'}, {'name': 'departure_time', 'type': 'STRING'}, {'name': 'arrival_time', 'type': 'STRING'}, {'name': 'departure_airport', 'type': 'STRING'}, {'name': 'arrival_airport', 'type': 'STRING'}, {'name': 'airline', 'type': 'STRING'}, {'name': 'lowest_price_adult', 'type': 'INTEGER'}]}
[2025-02-01T01:37:11.545+0000] {taskinstance.py:1350} INFO - Marking task as FAILED. dag_id=naver_flight_bigquery_pipeline, task_id=process_flight_data, execution_date=20250201T013009, start_date=20250201T013709, end_date=20250201T013711
[2025-02-01T01:37:11.555+0000] {standard_task_runner.py:109} ERROR - Failed to execute job 103 for task process_flight_data (Expected an instance of LoadJobConfig class for the job_config parameter, but received job_config = {'schema': [{'name': 'date', 'type': 'STRING'}, {'name': 'departure_time', 'type': 'STRING'}, {'name': 'arrival_time', 'type': 'STRING'}, {'name': 'departure_airport', 'type': 'STRING'}, {'name': 'arrival_airport', 'type': 'STRING'}, {'name': 'airline', 'type': 'STRING'}, {'name': 'lowest_price_adult', 'type': 'INTEGER'}]}; 461)
[2025-02-01T01:37:11.566+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 1
[2025-02-01T01:37:11.585+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
